persists/backend_server/Ballerina.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+org = "bff_samples"
+name = "persists"
```

Comment:
Think of a better name for this sample.

Reply:
something like `data persistence` would do

Reply:
updated.